gateway_code/tests/gateway_logging_test.py

```diff
@@ -23,13 +23,12 @@
 # pylint: disable=missing-docstring
 # pylint: disable=too-many-public-methods
 
-import os
 import tempfile
 import unittest
 import logging
+from StringIO import StringIO
```

Comment:
This seems unrelated but should be fixed after #18 I guess

Reply:
Yes, I've tested locally a merge and it seems to not create conflict